src/components/Category/Category.js

```diff
@@ -1,11 +1,37 @@
 import React from 'react';
+import './Category.css';
 
-function Category() {
-    return (
-      <>
-        <h2>Does this work</h2>
-      </>
-    );
-}
+const Category = ({ data }) => {
+  const filteredMakeupTypes = data.map(item => {
```

Comment:
If we're unable to refactor all our categories into one come Monday, I think we should delete this file and make mention of it as an `wontfix` issue for the MVP of the project.